plugins/modules/helm.py

```diff
@@ -413,6 +419,9 @@ def main():
     if not IMP_YAML:
         module.fail_json(msg=missing_required_lib("yaml"), exception=IMP_YAML_ERR)
 
+    if not HAS_TEMPFILE:
```

Comment:
two questions on this one
* is there a python version that we support that doesn't have tempfile?
* if there is, should we only check this when `atomic` is set to true?

Reply:
1. Oh. You are right, `tempfile` is already available. I will remove this code block.
2. This is unrelated to `atomic` parameter.